wp-includes/sqlite-ast/class-wp-sqlite-driver.php

```diff
@@ -699,9 +726,17 @@ private function execute_mysql_query( WP_Parser_Node $node ): void {
 			case 'createStatement':
 				$subtree = $node->get_first_child_node();
 				switch ( $subtree->rule_name ) {
+					case 'createDatabase':
+						/*
+						 * TODO:
+						 * This should probably be a no-op, in combination with
+						 * DROP DATABASE deleting the data file and recreating it.
+						 */
```

Comment:
We could handle it by creating a new `$slugified_db_name.sqlite` file. For now, let's throw a clear error. We should not see a CREATE DATABASE query in practice, but if some plugin uses it we'll need to know.

Reply:
:+1: At the moment, it will explicitly fail (fall trough to the `default` case), but I'll update the comment to add your idea.

Reply:
The reason I added this comment is that the old driver implements this as a no-op. Now, we're stricter, so let's see what it does in the test suite.